src/entity/EnemyShipFormation.java

```diff
@@ -314,7 +315,7 @@ public final void shoot(final Set<Bullet> bullets) {
         if (this.shootingCooldown.checkFinished()) {
             this.shootingCooldown.reset();
             bullets.add(BulletPool.getBullet(shooter.getPositionX()
-                + shooter.width / 2, shooter.getPositionY(), BULLET_SPEED, null, "ENEMY"));
+                + shooter.width / 2, shooter.getPositionY(), BULLET_SPEED, 1, "Down", "ENEMY"));
```

Comment:
```suggestion
                + shooter.width / 2, shooter.getPositionY(), BULLET_SPEED, 1, null, "ENEMY"));
```
안쓰이는 클래스이긴 하지만 굳이 수정하자면 모두 대문자로 "DOWN" 으로 하거나 null 값으로 해두면 될 것 같습니다.